.editorconfig

```diff
@@ -0,0 +1,35 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+# JavaScript files
+[*.{js,jsx,ts,tsx}]
+indent_style = space
+indent_size = 2
+quote_type = single
+
+# JSON files
+[*.json]
+indent_style = space
+indent_size = 2
+
+# CouchDB document property names
+[*.{couch,couchdb}]
+indent_style = space
+indent_size = 2
+
```

Comment:
Since we already have `indent_size = 2` and `indent_style = space` in the global section, I guess we don't need to re-specify this for other file-types, right?

```suggestion
```